<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form2</class>
 <widget class="QWidget" name="Form2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>463</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Module music information</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="QSlider" name="musicPos">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="spdLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spdBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="bpmBox">
        <property name="minimum">
         <number>32</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="bpmLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>440</width>
       <height>115</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamknij</string>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>30</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba sampli:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pattern:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>72</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba kanałów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>93</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rows in pattern:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>51</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba instrumentów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sampleLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="patternLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="rowLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="chanLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sampleButton">
      <property name="geometry">
       <rect>
        <x>244</x>
        <y>40</y>
        <width>101</width>
        <height>59</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Pokaż listę
instrumentów
i sampli</string>
      </property>
     </widget>
     <widget class="QLabel" name="instrLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Brak odtwarzanej muzyki</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="modformatLabel">
        <property name="text">
         <string>Format MOD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QFrame" name="okno"/>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>81</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>00 : 00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spdBox</tabstop>
  <tabstop>bpmBox</tabstop>
  <tabstop>sampleButton</tabstop>
  <tabstop>musicPos</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
